src/main/webapp/app/course/course-scores/course-scores.component.ts

```diff
@@ -224,7 +227,7 @@ export class CourseScoresComponent implements OnInit, OnDestroy {
                         console.warn('found more than one result for student ' + student.user.login + ' and exercise ' + exercise.title);
                     }
 
-                    const pointsAchievedByStudentInExercise = (result.score! * relevantMaxPoints) / 100;
+                    const pointsAchievedByStudentInExercise = round((result.score! * relevantMaxPoints) / 100, 1);
```

Comment:
Could we change the decimal places to two here? It seems that this would round quarter points to `.3` and quarter points are relatively common I would say.

Reply:
This decision was made with @krusche. If I understood it correctly, this is correct as we want to round on the individual exercise level. The decision was made to round to 1 decimal place here.

Reply:
I'm not a big fan of quarter points, in particular if we then would get issues with half points. We previously had issues that rounding to 2 places led to cases in which we suddenly had 2,99 or 3,01 points instead of 3,0

I would suggest we investigate this further in a follow-up PR in which we also write a lot of tests to make sure the behavior is always correct

Reply:
By the way: you can prevent quarter points by simply doubling the number of overall points in an exercise / exam. Then you can simply give half points.

In the future, we could let the instructor decide what kind of precision (in terms of decimal places) is desired in the course / exam and use this for rounding in all places.